From 723d67876783ffb2218514bf3bcfa7721ae21272 Mon Sep 17 00:00:00 2001
From: Colin Patrick Mccabe <cmccabe@cloudera.com>
Date: Mon, 2 May 2016 08:58:32 -0700
Subject: [PATCH 2011/2748] HADOOP-13072. WindowsGetSpaceUsed constructor
 should be public (Vinayakumar B via cmccabe)

(cherry picked from commit 2beedead72ee9efb69218aaf587de585158d6a1c)
(cherry picked from commit b16c376c3b62ec60aac4041099c4235bdaca815e)

Change-Id: Ieb649a428ac5d4210f44237f5fc3e229bac07a27
---
 .../org/apache/hadoop/fs/WindowsGetSpaceUsed.java  |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/WindowsGetSpaceUsed.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/WindowsGetSpaceUsed.java
index deb1343..2796089 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/WindowsGetSpaceUsed.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/WindowsGetSpaceUsed.java
@@ -32,7 +32,8 @@
 public class WindowsGetSpaceUsed extends CachingGetSpaceUsed {
 
 
-  WindowsGetSpaceUsed(CachingGetSpaceUsed.Builder builder) throws IOException {
+  public WindowsGetSpaceUsed(CachingGetSpaceUsed.Builder builder)
+      throws IOException {
     super(builder.getPath(), builder.getInterval(), builder.getInitialUsed());
   }
 
-- 
1.7.9.5

